waspc/examples/todoApp/src/client/user.ts

```diff
@@ -0,0 +1,33 @@
+import { User } from '@wasp/auth/types'
+import { findUserIdentity, getEmail } from '@wasp/auth/user'
+
+export function getName(user?: User) {
```

Comment:
Since `providerName` is a `string` at runtime, we can't really force type level checks and cause errors about some provider not being used.

Maybe as an extra improvement in the future, after we get some user feedback, we can do some compiler magic like use the knowledge about the enable auth method to do _something_ better with the username helpers. But I wouldn't expand the scope of this PR any more.